code/game/objects/items/stacks/rods.dm

```diff
@@ -9,7 +9,7 @@
 	throwforce = 15.0
 	throw_speed = 5
 	throw_range = 20
-	matter = list(DEFAULT_WALL_MATERIAL = 1875)
+	matter = list(DEFAULT_WALL_MATERIAL = SHEET_MATERIAL_AMOUNT / 2)

```

Comment:
It's a macro, so you should be planning around it potentially changing, either with comments on the macro or sanity stuff in the code. If someone changes it to an odd number for whatever reason it's going to end up a float and mess with the lathe.

BYOND assumes int until it needs to be a float I believe.